src/EditorFeatures/Core/Diagnostics/AbstractDiagnosticsAdornmentTaggerProvider.cs

```diff
@@ -32,15 +32,17 @@ internal abstract partial class AbstractDiagnosticsAdornmentTaggerProvider<TTag>
         protected sealed override bool IsEnabled => true;
 
         protected sealed override ITagSpan<TTag>? CreateTagSpan(
-            Workspace workspace, SnapshotSpan span, DiagnosticData data)
+            Workspace workspace, bool isLiveUpdate, SnapshotSpan span, DiagnosticData data)
```

Comment:
this flag was returned.  it is needed by push-diagnostics vs pull to fixup spans that might be incorrect in the pushed diagnostics vs the document as it exists right now.  Logic restored from: https://github.com/dotnet/roslyn/pull/65538/files